src/plugins/macaddr/README.md

```diff
@@ -0,0 +1,100 @@
+- infos = Information about the macaddr plugin is in keys below
+- infos/author = Thomas Bretterbauer <e01306821@student.tuwien.ac.at>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = check
+- infos/recommends =
+- infos/placements = postgetstorage presetstorage
+- infos/status = maintained nodep
+- infos/metadata = check/macaddr
+- infos/description = Validates MAC-addresses and returns them as integers
+
+## Introduction
+
+This plugin validates MAC-addresses. The following MAC-address-formats are supported:
+
+    XX-XX-XX-XX-XX-XX
+    XX:XX:XX:XX:XX:XX
+    XXXXXX-XXXXXX
+
+If `set` is used, the value will be transformed to the format of the middle one.
```

Comment:
Is this still true? I thought you used a decimal number now?

Reply:
You are right, this isn't true anymore. But I also do not transform anything anymore on `set`, just validating. `get` now transforms a given address to an integer representation.